go/cmd/dolt/dolt.go

```diff
@@ -561,7 +561,7 @@ func splitArgsOnSubCommand(args []string) (globalArgs, subArgs []string, initCli
 		if _, ok := commandSet[arg]; ok {
 			// SQL is the first subcommand to support the CLIContext. We'll need a more general solution when we add more.
 			// blame, table rm, and table mv commands also depend on the sql command, so they are also included here.
-			initCliContext := "sql" == arg || "blame" == arg || "table" == arg
+			initCliContext := "sql" == arg || "blame" == arg || "table" == arg || "status" == arg
```

Comment:
Talk to nick about this. I think he has something better on the way

Reply:
Nick's PR has been approved. Once that's merged, I'll rebase my changes and this should no longer be necessary.

Reply:
Rebase onto main. You'll get a conflict, but Nick's new ParseGlobalArgs method is better than my hack